misc/units.py

```diff
@@ -268,7 +268,7 @@ def run_shrink(cmdline_template, finput, foutput, lang):
 
     print('Shrinking ' + finput + ' as ' + lang)
     # fallback to the shell script version
-    subproces.run([SHELL, script, 'shrink',
+    subprocess.run([SHELL, script, 'shrink',
```

Comment:
@k-takata, could you look at this change for units.py?

Reply:
Ah, obviously it's my mistake. `subprocess` is correct.

Reply:
I just rebased onto this PR